src/_includes/layouts/post.njk

```diff
@@ -61,20 +74,20 @@ type: post
                 <div class="sticky top-4 mt-6 flex flex-col">
                     <h3 class="mb-3">Written By:</h3>
                     {% for author in authors %}
-                        {% if people[author] %}
-                            {% renderTeamMember people[author] %}
-                        {% endif %}
+                    {% if people[author] %}
+                    {% renderTeamMember people[author] %}
+                    {% endif %}
```

Comment:
Why? I liked it dented. This feels odd

Reply:
This was vs code, I'm reverting those now